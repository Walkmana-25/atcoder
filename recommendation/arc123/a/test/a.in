243273070809580 73791141625142 269557246614471