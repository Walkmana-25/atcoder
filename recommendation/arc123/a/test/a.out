182624017086885